html/semantics/links/links-created-by-a-and-area-elements/target_blank_implicit_noopener_base.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset=utf-8>
+  <title>Test behavior of base target=_blank links</title>
+  <script src=/resources/testharness.js></script>
+  <script src=/resources/testharnessreport.js></script>
+  <base target=_blank>
+</head>
+<body>
+  <a href="support/target_blank_iplicit_noopener.html?a1" id="a1" rel="noopener">Click me</a>
+  <a href="support/target_blank_iplicit_noopener.html?a2" id="a2" rel="opener">Click me</a>
+  <a href="support/target_blank_iplicit_noopener.html?a3" id="a3">Click me</a>
+  <a href="support/target_blank_iplicit_noopener.html?a4" id="a4" rel="opener noopener">Click me</a>
+  <a href="support/target_blank_iplicit_noopener.html?a5" id="a5" rel="noopener opener">Click me</a>
```

Comment:
Test interaction with noreferrer too?

The use of base target=_blank seems really subtle.

Reply:
Note that there's also a test without `<base>`. I figured we should test both.